refinery/core/api.py

```diff
@@ -678,24 +671,34 @@ def filter_by_group(self, request, obj_list):
         return obj_list
 
     def obj_get(self, bundle, **kwargs):
-        dataset = SharableResourceAPIInterface.obj_get(self, bundle, **kwargs)
-        return self.handle_incomplete_dataset(dataset)
+        try:
+            dataset = DataSet.objects.get(uuid=kwargs["uuid"])
```

Comment:
I'd replace this whole block with:
```python
dataset = super(ModelResource, self).obj_get(bundle, **kwargs)
```
and no exception handling necessary since this will raise `NotFound` if there are any problems